tests/queries/0_stateless/03236_squashing_high_memory.sql

```diff
@@ -0,0 +1,25 @@
+-- Tags: no-fasttest
+-- reason: test requires too many rows to read
+
+DROP TABLE IF EXISTS id_values;
+
+DROP TABLE IF EXISTS test_table;
+
+CREATE TABLE id_values ENGINE MergeTree ORDER BY id1 AS
+    SELECT arrayJoin(range(20000)) AS id1, arrayJoin(range(1000)) AS id2;
```

Comment:
If you reduce it too much the test loses its point
Please check if you can increase the row limit for this query or something else.